.ci/windows-sdk-facebook-vs2017.yml

```diff
@@ -4,244 +4,58 @@ pr:
       - master
   paths:
     include:
-      - .ci/windows-sdk-facebook.yml
+      - .ci/windows-sdk-facebook-vs2017.yml
       - .ci/templates/windows-sdk.yml
 trigger:
   branches:
     include:
       - master
   paths:
     include:
-      - .ci/windows-sdk-facebook.yml
+      - .ci/windows-sdk-facebook-vs2017.yml
       - .ci/templates/windows-sdk.yml
 jobs:
-  - job: windows_armv7
-    condition: false
-    pool: Facebook-VS2017
-    variables:
+  - template: templates/windows-sdk.yml
+    parameters:
+      VisualStudio: 2017/Community
+      pool: Facebook-VS2017
       branch: master
+      toolchain: 5
```

Comment:
I am sure everyone might appreciate a textual comment about which toolchain IDs are being used. I have to look in the template how it was expanded, but something like `toolchain_pipeline_id` or just `toolchain_pipeline` might had help.

Reply:
I like the comment idea!  I'll actually add that in a follow up change (the testing cycle here is pretty insane)